### PR TITLE
SKIT-77 : Dapp Store Schema

### PR DESCRIPTION
**Pull request by rishikesh-y**
I have tested it locally with StoreKit and it's working fine.

### CODE REVIEW

File: src/dappStore.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+                    "org.uniswap.dapp",
+                    "io.opensea.dapp"
+                ],
+                "description": "A curated list of dapps that we think you will love.",
+                "key": "curated-list"
+            }],
+            "key": "reeshi.dappstore"
+        }
+    ]
+}
```
Comment (r4881t) [inline src/dappStore.json:17]: extra line at end of file required.

**Review by r4881t — CHANGES_REQUESTED** [review]

File: src/dappStore.json (hunk 1 above)
Comment (rishikesh-y) [reply src/dappStore.json]: done

File: src/lib/utils.ts [hunk 2]
```diff
@@ -22,16 +42,261 @@ export class cloneable {
...
+  commitMessage: string,
+  org: string | undefined = undefined,
+  githubOwner: string,
+  githubRepo: string,
+  schema: string
+) => {
+  let File: string;
+  if (schema === "registry") {
+    File = "src/registry.json";
+  } else {
```
Comment (kailash-bitpack) [inline src/lib/utils.ts:210]: const file = schema === "registry" ? "src/registy.json" : "src/...."

File: src/lib/utils.ts [hunk 3]
```diff
@@ -22,16 +42,261 @@ export class cloneable {
...
+
+/**
+ * It will check that dapps exist in registry or not.
+ */
+export const isExistInRegistry = async (
+  dappIds: string[],
+  DappRegistry: DappStoreRegistry
+) => {
+  const currRegistry = await DappRegistry.registry();
+  dappIds.map(x => {
```
Comment (kailash-bitpack) [inline src/lib/utils.ts:276]: use .forEach

File: src/lib/utils.ts (hunk 2 above)
Comment (rishikesh-y) [reply src/lib/utils.ts]: done

File: src/lib/utils.ts (hunk 3 above)
Comment (rishikesh-y) [reply src/lib/utils.ts]: done

**Review by kailash-bitpack — APPROVED** [review]

**Pull request merged**